src/components/Navbar.vue

```
@@ -0,0 +1,29 @@
+<script lang="ts" setup>
```

ESLint

[error] Component name "Navbar" should always be multi-word.

(vue/multi-word-component-names)